src/services/EndPoints.tsx

```diff
@@ -3,7 +3,7 @@ import axios from "axios"
 const baseURL = 'https://upskilling-egypt.com:3003/api/v1'
 export const imageBaseURL = 'https://upskilling-egypt.com:3003'
 
-export const axiosInstanceUrl = axios.create({baseURL: baseURL})
+export const axiosInstanceURL = axios.create({baseURL: baseURL})
 
 export const axiosInstance = axios.create({baseURL: baseURL, headers: {
```

Comment:
We need to create a private one as well to avoid sending the token unnecessarily.